spec/pom.xml

```diff
@@ -62,11 +61,11 @@
     <build>
         <defaultGoal>package</defaultGoal>
         <plugins>
-            <!-- Sets minimal Maven version to 3.5.4 and Java version to 11 -->
+            <!-- Sets minimal Maven version to 3.6.0 and Java version to 11 -->
```

Comment:
Since we're removing the java 11 aspects of this plugin let's update the comment as was done in the other poms.

Reply:
Okay, it is done.